.pre-commit-config.yaml

```diff
@@ -1,13 +1,13 @@
 fail_fast: true
 
 repos:
-  - repo: local
```

Comment:
I had to disable ruff due to an issue with yaml files which I can't understand :|

Reply:
In the end I put ruff working only for .py files, I checked on master and the error on yaml is there :/